src/main/java/com/github/chabanenk0/Service/CloneCreator.java

```diff
@@ -0,0 +1,73 @@
+package com.github.chabanenk0.Service;
+
+import com.github.chabanenk0.Annotation.Clone;
+
+import java.lang.Object;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+
+/**
+ * Created by dmitry on 21.12.16.
+ */
+public class CloneCreator
+{
+    public Object clone(Object sourceObject) throws IllegalAccessException, InstantiationException {
+        Class classObject = sourceObject.getClass();
+        Object newObject = classObject.newInstance();
+        Field[] fields = classObject.getDeclaredFields();
+        for (Field field : fields) {
+            String fieldName = field.getName();
```

Comment:
a bit on deep cloning and serialization: http://javatechniques.com/blog/faster-deep-copies-of-java-objects/